src/app.rs

```diff
@@ -8,18 +8,10 @@ use std::collections::BTreeMap;
 use self::clap::{App, Arg, ArgGroup, AppSettings};
 use super::{Slurp, RunOptions, compile, DEFAULT_SHELL, maybe_s};
 
-macro_rules! warn {
-  ($($arg:tt)*) => {{
-    extern crate std;
-    use std::io::prelude::*;
-    let _ = writeln!(&mut std::io::stderr(), $($arg)*);
-  }};
-}
-
 macro_rules! die {
   ($($arg:tt)*) => {{
     extern crate std;
```

Comment:
Why do you need the `extern crate std` if you're already relying on `::std::process` being in scope?